src/read_agent_logging.in

```diff
@@ -20,22 +20,32 @@ URL="${1:-http://localhost:24231/metrics}"
 INTERVAL=${COLLECTD_INTERVAL:-10}
 
 # The part of the input that sed looks at looks like:
-#   stackdriver_successful_requests_count{grpc="false"} 255
-#   stackdriver_failed_requests_count{grpc="false",code="400"} 14
+# 1. If the Logging Agent uses REST.
+#   stackdriver_successful_requests_count{grpc="false",code="200"} 255
 #   stackdriver_failed_requests_count{grpc="false",code="404"} 18
-#   stackdriver_ingested_entries_count 7050
-#   stackdriver_dropped_entries_count 25
-#   stackdriver_retried_entries_count{code="500"} 2
-#   stackdriver_retried_entries_count{code="501"} 3
-#
+#   stackdriver_ingested_entries_count{grpc="false",code="200"} 7050
+#   stackdriver_dropped_entries_count{grpc="false",code="401"} 25
+#   stackdriver_retried_entries_count{grpc="false",code="500"} 2
+# 2. If the Logging Agent uses gRPC.
+#   stackdriver_successful_requests_count{grpc="true",code="0"} 255
+#   stackdriver_failed_requests_count{grpc="true",code="3"} 18
+#   stackdriver_ingested_entries_count{grpc="true",code="0"} 7050
+#   stackdriver_dropped_entries_count{grpc="true",code="3"} 25
+#   stackdriver_retried_entries_count{grpc="true",code="13"} 2
+
 # The expected output follows:
+# 1. If the Logging Agent uses REST.
```

Comment:
It might be easier to read if you kept the sample inputs and outputs for the REST path together, and the same for the gRPC path.

Reply:
Changed.